### PR TITLE
Issue #SB-16917 fix: Post signup content starts to play in the backgr…

### PR DESCRIPTION
**Pull request by ArjunKhetia-NIIT**
…ound before completing the onboarding details

### CODE REVIEW

File: src/app/client/src/app/service/popup-control.service.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  public showUserVerificationPopup = false;
+  public isLocationConfirmed = true;
+
+  public popupSource = new BehaviorSubject(null);
+  checkPopupStatus = this.popupSource.asObservable();
+
+  constructor() { }
+
+  changePopupStatus(modalkey: string, modalvalue: boolean) {
+    if (modalkey === 'showFrameWorkPopUp') {
```
Comment (rajeevsathish) [inline src/app/client/src/app/service/popup-control.service.ts]: why cannot this be a switch

File: src/app/client/src/app/app.component.ts [hunk 2]
```diff
@@ -195,10 +196,12 @@ export class AppComponent implements OnInit, OnDestroy {
           if (!deviceProfile.userDeclaredLocation ||
             !(this.usersProfile && this.usersProfile.userLocations && this.usersProfile.userLocations.length >= 1)) {
             this.isLocationConfirmed = false;
+            this.popupControlService.changePopupStatus('isLocationConfirmed', false);
```
Comment (rajeevsathish) [inline src/app/client/src/app/app.component.ts]: the if and else are doing the same call why do you want to call it 2times

File: src/app/client/src/app/service/popup-control.service.ts (hunk 1 above)
Comment (ArjunKhetia-NIIT) [reply src/app/client/src/app/service/popup-control.service.ts]: Done..

File: src/app/client/src/app/app.component.ts (hunk 2 above)
Comment (ArjunKhetia-NIIT) [reply src/app/client/src/app/app.component.ts]: Sir That If-Else is already been written. I have No idea about it.

**Pull request merged**